.github/workflows/build_and_test.yml

```diff
@@ -318,36 +318,21 @@ jobs:
   #       cat "$GITHUB_WORKSPACE/OpenDDS/${{ github.job }}_autobuild_workspace/latest.txt"
   #       if ! grep -q 'Failures: 0' "$GITHUB_WORKSPACE/OpenDDS/${{ github.job }}_autobuild_workspace/latest.txt"; then echo "::error file=latest.txt,line=0,col=0::Test Failures: see 'Test Results: ${{ github.job }}'"; fi
 
-  ACE_TAO_u18_o1d0v1_xer0_j12_2:
+  ACE_TAO_u18_o1d0v1_xer0_j12:
 
     runs-on: ubuntu-18.04
 
-    needs: wait_cancel_previous
-
     steps:
     - name: checkout OpenDDS
       uses: actions/checkout@v2.3.4
       with:
         path: OpenDDS
         submodules: true
-    - name: checkout ACE_TAO
+    - name: checkout autobuild
       uses: actions/checkout@v2.3.4
       with:
-        repository: DOCGroup/ACE_TAO
-        ref: ace6tao2
-        path: OpenDDS/ACE_TAO
-    - name: get ACE_TAO commit
-      shell: bash
-      run: |
-        cd OpenDDS/ACE_TAO
-        export ACE_COMMIT=$(git rev-parse HEAD)
-        echo "ACE_COMMIT=$ACE_COMMIT" >> $GITHUB_ENV
-    - name: Cache Artifact
-      id: cache-artifact
-      uses: actions/cache@v2.1.4
-      with:
-        path: ${{ github.job }}.tar.xz
-        key: ${{ github.job }}_ace6tao2_${{ env.ACE_COMMIT }}
```

Comment:
Why does this change remove caching of ACE_TAO?  Seems like that would result in longer build times

Reply:
It doesn't build a seperate ace anymore.  The separate ace was not properly being brought into the make installed directory.

Reply:
This job should probably get a new name then, since it's not just an ACE_TAO job?
Also, we can remove all the "if: steps.cache-artifact.outputs.cache-hit != 'true'" checks below since we're not caching anything.
And I agree that no caching is sad times... our total build times on GHA are starting to creep upward.

Reply:
There is nothing inherent in `make install` that would prevent caching.  We essentially have this with Jenkins where ACE_TAO jobs are distinct from (and used by) OpenDDS jobs.